Essentials/src/com/earth2me/essentials/commands/Commandeco.java

```diff
@@ -91,10 +104,14 @@ private void set(BigDecimal amount, final User player, final CommandSource sende
         BigDecimal maxBalance = ess.getSettings().getMaxMoney();
         boolean underMinimum = (amount.compareTo(minBalance) < 0);
         boolean aboveMax = (amount.compareTo(maxBalance) > 0);
-        player.setMoney(underMinimum ? minBalance : aboveMax ? maxBalance : amount, UserBalanceUpdateEvent.Cause.COMMAND_ECO);
-        player.sendMessage(tl("setBal", NumberUtil.displayCurrency(player.getMoney(), ess)));
-        if (sender != null) {
-            sender.sendMessage(tl("setBalOthers", player.getDisplayName(), NumberUtil.displayCurrency(player.getMoney(), ess)));
+        if (isPercent) {
+            player.percentHandler(amount, sender, UserBalanceUpdateEvent.Cause.COMMAND_ECO, 2);
```

Comment:
This implementation doesn't line up with the implementation below... I get its non-congruent with the above two methods but this way makes it look like the messages are missing.